lore-background.md

```diff
@@ -66,6 +67,7 @@ Note: Canon date of game start is in "YEAR", which equals the IRL year the game
 - **December 18**: Sample XE-037 is brought back to Earth from the netherum. It is about thirty grams of blob matter. Standard decontamination protocols are observed. However, standard protocols do not take into account the possibility of malicious, intelligent molecular-scale contaminants. Microscopic particles of XE-037 migrate from the contamination filters onto the skin of scientists and escape containment. Soon after, the Blob enters the groundwater and begins to spread at extremely low concentrations.
 
 #### Year -1
+- **March**: The first effects of the blob contamination become visible as bees two or even three times their size are sighted. They are non-aggressive and appear to cause more good than harm due to increased honey production.
```

Comment:
I don't think bigger bees would make more honeys, actually bigger bees would probably need more pollen which would either be impossible to aquire enough to sustain themselves or could lead to a significant decrease in biodiversity as the giant bees overcompete every other pollinating insects.

Reply:
That is actually meant as an unreliable narration. Bigger bees would be terrible.

Reply:
So not a thing to inscribe in the design doc as being actually true